<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1172</width>
    <height>734</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>160</y>
      <width>851</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>17</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>SearchSerial</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exploreButton">
         <property name="text">
          <string>Explore</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="searchLE"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Search</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="CustomTextBrowser" name="searchTB"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>17</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Add Serial</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Year</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Seasons</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLineEdit" name="serialName"/>
           </item>
           <item>
            <widget class="QLineEdit" name="serialYear"/>
           </item>
           <item>
            <widget class="QLineEdit" name="serialSize"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="CustomTextBrowser" name="addTB"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1172</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomTextBrowser</class>
   <extends>QTextBrowser</extends>
   <header>customtextbrowser.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>Search()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>494</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>547</x>
     <y>496</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>AddData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>727</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>1053</x>
     <y>504</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exploreButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>Explore()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>48</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>Clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>505</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>574</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Search()</slot>
  <slot>AddData()</slot>
  <slot>Explore()</slot>
  <slot>Clear()</slot>
 </slots>
</ui>
